var/spack/repos/builtin/packages/libaio/package.py

```diff
@@ -3,22 +3,23 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
-
 
 class Libaio(Package):
```

Comment:
Can you convert this to `MakefilePackage`?

Reply:
Done